src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/PayPalExpressToQuoteMapper.php

```diff
@@ -0,0 +1,105 @@
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function mapAddressTransfer(
+        QuoteTransfer $quoteTransfer,
+        ComputopPayPalExpressInitResponseTransfer $computopPayPalExpressInitResponseTransfer
+    ): QuoteTransfer {
+        $shippingAddressTransfer = new AddressTransfer();
+        $countryTransfer = new CountryTransfer();
+        $countryTransfer->setIso2Code($computopPayPalExpressInitResponseTransfer->getAddressCountryCode());
+        $shippingAddressTransfer->setFirstName($computopPayPalExpressInitResponseTransfer->getFirstName());
+        $shippingAddressTransfer->setLastName($computopPayPalExpressInitResponseTransfer->getLastName());
+        $shippingAddressTransfer->setAddress1($computopPayPalExpressInitResponseTransfer->getAddressStreet());
+        $shippingAddressTransfer->setAddress2($computopPayPalExpressInitResponseTransfer->getAddressStreet2());
+        $shippingAddressTransfer->setCity($computopPayPalExpressInitResponseTransfer->getAddressCity());
+        $shippingAddressTransfer->setState($computopPayPalExpressInitResponseTransfer->getAddressState());
+        $shippingAddressTransfer->setZipCode($computopPayPalExpressInitResponseTransfer->getAddressZip());
+        $shippingAddressTransfer->setIso2Code($computopPayPalExpressInitResponseTransfer->getAddressCountryCode());
+        $shippingAddressTransfer->setCountry($countryTransfer);
+        $shippingAddressTransfer->setPhone($computopPayPalExpressInitResponseTransfer->getPhone());
+
+        foreach ($quoteTransfer->getItems() as $itemTransfer) {
+            $itemTransfer->getShipment()->setShippingAddress($shippingAddressTransfer);
+        }
+
+        foreach ($quoteTransfer->getExpenses() as $expenseTransfer) {
+            $expenseTransfer->getShipment()->setShippingAddress($shippingAddressTransfer);
+        }
+
+        $quoteTransfer->setShippingAddress($shippingAddressTransfer);
+
+        return $quoteTransfer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer $computopPayPalExpressInitResponseTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function mapBillingTransfer(
+        QuoteTransfer $quoteTransfer,
+        ComputopPayPalExpressInitResponseTransfer $computopPayPalExpressInitResponseTransfer
+    ): QuoteTransfer {
+        $billingAddressTransfer = new AddressTransfer();
+        $countryTransfer = new CountryTransfer();
+        $countryTransfer->setIso2Code($computopPayPalExpressInitResponseTransfer->getAddressCountryCode());
+        $billingAddressTransfer->setFirstName($computopPayPalExpressInitResponseTransfer->getFirstName());
+        $billingAddressTransfer->setLastName($computopPayPalExpressInitResponseTransfer->getLastName());
```

Comment:
can we use `$billingAddressTransfer->fromArray($computopPayPalExpressInitResponseTransfer->toArray())` and add custom mapping after?
several fields are the same